### DISCUSSION TITLE
Set as base run

### DISCUSSION THREAD

**Discussion by MoritzSalein** [ludeknovy/jtl-reporter#120] [Q&A] [answered]
What can I do with the option "Set as base run" in the test run edit dialogue?

**Comment by ludeknovy** [comment, marked as answer]
Hi! 

It marks the given test report as a reference, it was intended to be used once you are happy with the SUT performance. And later on could easily compare the other reports to it. If you have the base run in a scenario, you will see "compare to base run" option in stats menu. Upon clicking it will show you the comparison.

![Screenshot 2022-02-01 at 11 55 04](https://user-images.githubusercontent.com/13610612/151955987-4eb97503-5f08-4a16-be60-c095086d68a3.png)
